cmd/ostracon/commands/show_validator.go

```diff
@@ -15,18 +18,29 @@ var ShowValidatorCmd = &cobra.Command{
 	Use:     "show-validator",
 	Aliases: []string{"show_validator"},
 	Short:   "Show this node's validator info",
-	RunE:    showValidator,
-	PreRun:  deprecateSnakeCase,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return showValidator(cmd, args, config)
+	},
+	PreRun: deprecateSnakeCase,
 }
 
-func showValidator(cmd *cobra.Command, args []string) error {
-	keyFilePath := config.PrivValidatorKeyFile()
-	if !tmos.FileExists(keyFilePath) {
-		return fmt.Errorf("private validator file %s does not exist", keyFilePath)
+func showValidator(cmd *cobra.Command, args []string, config *cfg.Config) error {
+	var pv types.PrivValidator
+	var err error
+	if config.PrivValidatorListenAddr != "" {
+		chainID := "" // currently not in use
+		pv, err = node.CreateAndStartPrivValidatorSocketClient(config.PrivValidatorListenAddr, chainID, logger)
```

Comment:
`config.ChainID()` isn't set anywhere except in testing, and I thought to avoid it because `chainID := config.ChainID()` could be a confusing bug that looks like correct code but is never actually the correct value.

Currently, neither ostracon-kms nor tmkms use `chainID` to identify the `PrivValidator`, so I assumed `""`, but I'll rewrite it to use the correct `chainID` according to [node.go(759)](https://github.com/line/ostracon/blob/7cb986b6492f480847caa2bab317d020da60d6ca/node/node.go#L759)'s method.